calendarScript.js

```diff
@@ -0,0 +1,74 @@
+const date = new Date();
+
+const renderCalendar = () => {
+
+    date.setDate(1);
+
+    const monthDays = document.querySelector(".days");
+
+    const lastDay = new Date(date.getFullYear(),
+    date.getMonth() + 1, 0).getDate();  // 0 = last day of pre month, 1 = first day of current month //
+
+    const prevLastDay = new Date(date.getFullYear(),
+    date.getMonth(), 0).getDate();
+
+    const firstDayIndex = date.getDay();
+    const lastDayIndex = new Date(date.getFullYear(),
+    date.getMonth() + 1, 0).getDay();
+
+    const nextDays = 7 - lastDayIndex - 1;
+
+
+    const months = [
+        "January",
+        "February",
+        "March",
+        "April",
+        "May",
+        "June",
+        "July",
+        "August",
+        "September",
+        "October",
+        "November",
+        "December"
+    ];
+
+    document.querySelector(".date h1").innerHTML = months[date.getMonth()];
+
+    document.querySelector(".date p").innerHTML = new Date().toDateString();
+
+    let days ="";
+
+    for(let x = firstDayIndex; x >0; x--) {
+        days += `<div class = "prev-date">${prevLastDay - x + 1}</div>`
+    }
+
+
+    for(let i = 1; i <= lastDay; i++) {
+        if(i === new Date().getDate() && date.getMonth() === new Date().getMonth())
+            {days += `<div>${i}</div>`; }
+        else {
+            days += `<div>${i}</div>`;
+        }
+    }
+
+    for(let i = 1; i <= nextDays; i++) {
+        days += `<div class = "next-date">${i}</div>`;
+        monthDays.innerHTML = days;
+    }
+
```

Comment:
```suggestion
    }
    
    monthDays.innerHTML = days;
```

`days`에 담아둔 요소들을 DOM에 적용하는 로직은 for문 밖, 즉 `renderCalendar` 함수의 제일 마지막에 한 번만 동작해도 될 것 같습니다.